dbinspector/src/main/java/im/dino/dbinspector/fragments/DatabaseListFragment.java

```diff
@@ -21,7 +21,9 @@
  * Created by dino on 23/02/14.
  */
 @TargetApi(Build.VERSION_CODES.HONEYCOMB)
```

Comment:
Have kept it coz did not got much time to test it in ginger bread device.
As per Target API documentation I don't think it can become problematic
unless we add any code that goes above our target API value
On 16-Feb-2015 12:53 pm, "Dino Kovač" <notifications@github.com> wrote:

> In
> dbinspector/src/main/java/im/dino/dbinspector/fragments/DatabaseListFragment.java
> <https://github.com/infinum/android_dbinspector/pull/9#discussion_r24734720>
> :
>
> > @@ -21,7 +21,9 @@
> >   * Created by dino on 23/02/14.
> >   */
> >  @TargetApi(Build.VERSION_CODES.HONEYCOMB)
>
> I hope we can get rid of this now. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/infinum/android_dbinspector/pull/9/files#r24734720>.
>